crt_portal/static/js/set_referrer_on_report.js

```diff
@@ -0,0 +1,19 @@
+(function(root) {
+  referrer = document.referrer;
+  host = document.location.host;
+  if (referrer.indexOf(host) > -1) {
+    referrer = '';
+  }
+
+  if (localStorage.getItem('referrer') == null && referrer != '') {
```

Comment:
Same as above.  Should we overwrite with the new referrer?

Reply:
I don't think so.  If a user is referred to the portal landing page from `SiteA`, and they then click to the form from the portal landing page, we want to record `SiteA` as the referrer.  In this scenario, overwriting local storage with the new referrer would cause us to save the portal landing page as the referrer.